tests/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -211,7 +224,8 @@ TEST (nnstreamer_capi_valve, test01)
   EXPECT_NE (state, ML_PIPELINE_STATE_UNKNOWN);
   EXPECT_NE (state, ML_PIPELINE_STATE_NULL);
 
-  g_usleep (150000); /* 150ms. Let a few frames flow. */
+  wait_for_start (handle, state, status);
```

Comment:
`wait_for_start` should be after `g_usleep`? as the state can be `ML_PIPELINE_STATE_READY` at this moment.

Reply:
How about `wait_for_start` loops till `ML_PIPELINE_STATE_READY < state < ML_PIPELINE_STATE_PAUSED`?
This would eliminate the need for extra `g_usleep` at all the places where `wait_for_start` is called.

Reply:
Changed applied.